test/tools/obdscanner.uts

```diff
@@ -225,8 +225,7 @@ responder.start()
 
 
 = Test DTC scan
-eprint("TODO: this test is currently failing. It has to be reworked")
-exit(0)
+
```

Comment:
This test is failing due to this change. Could you revert your modification?

Reply:
I am still trying to fix that test.
I did another commit that should fix this test without ignoring the test.